# joint_pose:
#   panda_joint1: 0
#   panda_joint2: -0.785398163397
#   panda_joint3: 0
#   panda_joint4: -2.35619449019
#   panda_joint5: 0
#   panda_joint6: 1.57079632679
#   panda_joint7: 0.785398163397

joint_pose:
  panda_joint1: 0.004782319344623101
  panda_joint2: -0.9558349683839263
  panda_joint3: 0.017065873309418004
  panda_joint4: -2.5405308878056743
  panda_joint5: 0.07942621420585264
  panda_joint6: 2.0632984707380797
  panda_joint7: 0.7367900029834772
